specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json

```diff
@@ -821,6 +918,69 @@
         }
       },
       "description": "Details needed to monitor a Hana Instance"
+    },
+    "SapMonitor": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SapMonitorProperties",
+          "description": "SAP monitor properties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "SAP monitor info on Azure (ARM properties and SAP monitor properties)"
+    },
+    "SapMonitorProperties": {
+      "properties": {
+        "hanaSubnet": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Specifies the SAP monitor unique ID."
+        },
+        "hanaHostname": {
+          "type": "string",
+          "description": "Hostname of the HANA instance."
+        },
+        "hanaDbName": {
+          "type": "string",
+          "description": "Database name of the HANA instance."
+        },
+        "hanaDbSqlPort": {
+          "type": "integer",
+          "description": "Database port of the HANA instance."
+        },
+        "hanaDbUsername": {
+          "type": "string",
+          "description": "Database username of the HANA instance."
+        },
+        "hanaDbPassword": {
+          "type": "string",
+          "description": "Database password of the HANA instance."
+        },
```

Comment:
Will this ever be returned on GET responses?

Reply:
No, it won't.
On the GET, will not exist.

Is there a way to reuse the definition with slight changes?